### PR TITLE
#165876175 Add navigation pills to header

### PR DESCRIPTION
**Pull request by koechkevin**
#### What does this PR do?
- Adds navigation pills to switch between automations and dashboard pages
#### Description of Task to be completed?
 - add a route to redirect to the dashboard
 - add links to the dashboard and navigation on the navbar
#### How should this be manually tested?
1. On browser, goto url `/`. 
2. On the nav bar click on the pills to navigate between dashboard and automations

#### Any background context you want to provide?
- NA
#### What are the relevant pivotal tracker stories?
[#165876175](https://www.pivotaltracker.com/story/show/165876175)
#### Postman Documentation
- N/A
#### Screenshots (If applicable)
Screenshot 2019-05-14 at 11 34 02

### CODE REVIEW

File: src/__tests__/components/Dashboard/index.test.js [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+    },
+  };
+  const mockStore = configureStore();
+  const store = mockStore(state);
   it('renders a card', () => {
-    // eslint-disable-next-line react/jsx-filename-extension
-    const wrapper = mount(<Dashboard />);
-    expect(wrapper).toMatchSnapshot();
-    wrapper.unmount();
+    const wrapper = mount(
```
Comment (FelixWambiri) [inline src/__tests__/components/Dashboard/index.test.js:26]: Nice work but I do not think this test really tests for rendering a card. There is no you have asserted that a card is displayed. I also think it is not necessary to pass state or props to this component more so when you omit them the test still passes.

Comment (koechkevin) [reply src/__tests__/components/Dashboard/index.test.js:26]: It actually does not test the rendering. This is a container for housing the other cards. The test will keep changing as the cards are added

File: src/components/Dashboard/index.jsx [hunk 2]
```diff
@@ -11,15 +14,21 @@ class Dashboard extends React.Component {
   You will create a function renderUpselling that returns
```
Comment (FelixWambiri) [inline src/components/Dashboard/index.jsx:14]: Nice explanation and straight forward steps on what the function you pass should do. Please edit the comments before this last statement as they are not clear. I think that what you meant is that the reusable card can be used to render different information as such three properties should be passed to it that is class, title and a function that returns jsx.

**Review by mikenthiwa — APPROVED** [review]
LGTM

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

**Pull request merged**